lib/config/defaults.js

```diff
@@ -143,10 +143,10 @@ function commit() {
  */
 function readFile(filePath, field) {
   if (filePath && fs.existsSync(filePath)) {
-    var data = fs.readFileSync(filePath);
+    var data = fs.readFileSync(filePath).toString();
     try {
       var json = JSON.parse(data)
-      return field ? (json[field] || {}) : field;
+      return field ? (json[field] || {}) : json;
     } catch(error) {
```

Comment:
Fix bug where the `server` and `watch` tasks did not respect settings in `angular.json`.